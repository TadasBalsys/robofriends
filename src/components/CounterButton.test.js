import { shallow } from "enzyme";
import React from "react";
import CounterButton from "./CounterButton";

// const wrapper = shallow();

// console.log(wrapper(<CounterButton />));

describe("CounteButton testing", () => {
  it("expect to render CounterButton component", () => {
    const mackColor = "red";
    expect(shallow(<CounterButton color={mackColor} />)).toMatchSnapshot();
  });
  it("correctly increments the counter", () => {
    const mackColor = "red";
    const wrapper = shallow(<CounterButton color={mackColor} />);

    wrapper.find('[id="counter"]').simulate('click');
    expect(wrapper.state()).toEqual({count: 1});
    expect(wrapper.props().color).toEqual("red");

  });
});
