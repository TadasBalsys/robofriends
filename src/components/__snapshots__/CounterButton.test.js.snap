// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CounteButton testing expect to render CounterButton component 1`] = `ShallowWrapper {}`;
